tests/pathfinding/test_api.py

```diff
@@ -162,7 +162,7 @@ def request_path_with(status_code=400, **kwargs):
 
     # bad signature
     bad_iou_dict = good_iou_dict.copy()
-    bad_iou_dict["signature"] = hex(int(bad_iou_dict["signature"], 16) + 1)
+    bad_iou_dict["signature"] = "0x" + "1" * 130
```

Comment:
Why did this start to bite us again?

Reply:
Because it was used in two different places.